CMake/fletch-tarballs.cmake

```diff
@@ -393,7 +393,32 @@ set(Darknet_url "https://data.kitware.com/api/v1/file/598866018d777f7d33e9c0f3/d
 set(Darknet_md5 "eea312b07685eef9c6f26cf63346fe41")
 list(APPEND fletch_external_sources Darknet)
 
+# PostgreSQL
```

Comment:
Take a look at one of the other versionable packages and make this pattern match. This one was done first and the others in fletch have since improved on this clunkiness, e.g. this one shows the PG version before the user actually selects it, which is not desirable.